### PR TITLE
Ready: Adding documentation for OpenData sources

### PR DESCRIPTION
**Pull request by speedyturkey**
This PR contains new documentation for several data sources (all from DC OpenData) as part of the effort led by @emkap01.

OpenData.md has general information on the DC OpenData initiative, and each source has its own .md file that links back to it. I'm new to the way internal markdown links work, so I am hopeful that I did them correctly.

Hope this is helpful! Happy to expand further as needed.

### CODE REVIEW

File: docs/data/building_permits.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
```
Comment (NealHumphrey) [inline docs/data/building_permits.md]: @speedyturkey I think that this yml frontmatter will be invalid. There should just be one pair of `---`, like this:

```
---
layout: datasource
tablename: building_permits
title: Building Permit Data
---
```

File: docs/data/opendata.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+A number of data sources come from the [opendata.dc.gov](http://opendata.dc.gov/) project. The OpenData project describes itself as a site where:
+
+> ...the District of Columbia government shares hundreds of datasets. DC realizes that data’s greatest value comes from having it freely shared among agencies, federal and regional governments and with the public to the extent possible when considering safety, privacy and security. 
+
+Users are invited to analyze the available datasets and build apps using OpenData's APIs.
+
+We have used the following data sources from OpenData. Please click the links below to read about each source in further detail.
+
+[Building Permits]({{site.baseurl}}/building_permits)
```
Comment (NealHumphrey) [inline docs/data/opendata.md]: Almost there. `{{site.baseurl}}` matches with the 'docs' folder when this is deployed. `docs` becomes `housinginsights.org`. So to get these urls to work correctly, you need to add the /data folder to the path i.e. `{{site.baseurl}}/data/building_permits`

File: docs/data/tax.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+title: DC Tax Data
+---
+
+This is the public extract of the database used for sending tax bills, also known as tax assessment roll.  ITS stores comprehensive tax information such as ownership, mailing addresses etc of record lots, tax lots, parcels, condominiums and federally owned lands such as reservations and appropriations. This dataset is owned and maintained by the DC Office of Tax and Revenue.
+
+This data is released on an annual basis - we display data from 2017. The latest data can be found at [OpenData DC](https://opendata.arcgis.com/datasets/DCGIS::integrated-tax-system-public-extract).
+
+Basic cleaning was performed on this data, including replacement of null values and normalization of dates.
+
+This data source is used with permission from [OpenData DC]({{site.baseurl}}/opendata).
```
Comment (NealHumphrey) [inline docs/data/tax.md]: update link

File: docs/data/crime.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+title: Crime Data
+---
+
+The DC Metropolitan Police have made this data available through their Crime Map service.  This dataset contains incidents of crime reported to the police.
+
+This data is released on an annual basis - we display data from 2015-2017. The latest data for 2017 can be found at [OpenData DC](http://opendata.dc.gov/datasets/crime-incidents-in-2017).
+
+Basic cleaning was performed on this data, including replacement of null values and normalization of dates and census tract/ward names.
+
+This data source is used with permission from [OpenData DC]({{site.baseurl}}/opendata).
```
Comment (NealHumphrey) [inline docs/data/crime.md]: update link, see other comment

File: docs/data/building_permits.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+title: Building Permit Data
+---
+
+The DC Department of Consumer and Regulatory Affairs has made this data available. This dataset contains records of building construction and alteration permits.
+
+This data is released on an annual basis - we display data from 2013-2017. The latest data for 2017 (partial) can be found at [OpenData DC](http://opendata.dc.gov/datasets/building-permits-in-2017).
+
+Basic cleaning was performed on this data, including replacement of null values and normalization of dates and census tract/ward names.
+
+This data source is used with permission from [OpenData DC]({{site.baseurl}}/opendata).
```
Comment (NealHumphrey) [inline docs/data/building_permits.md]: update link, see other comment

File: docs/data/building_permits.md (hunk 1 above)
Comment (NealHumphrey) [inline docs/data/building_permits.md]: it looks like all your files have 3 lines of `---` instead of just 2 (opening and closing)

**Comment by NealHumphrey** [comment]
@speedyturkey I requested a couple quick changes and therefore renamed this pull request. When you make the changes, rename it back to `Ready` and I'll take another look.

**Comment by speedyturkey** [comment]
Thanks for the feedback! Sorry for the silly errors, I know you have better things to do, but appreciated all the same.

**Comment by NealHumphrey** [comment]
@speedyturkey not a problem! I realized that the yaml frontmatter problem came from my example that was wrong, and there's always a barrier learning a new platform like Jekyll.

**Pull request merged**